app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -9,4 +10,53 @@ def contention_list
   def va_representative
     object.data['poa']
   end
+
+  def updates
+    updates = [
+      make_string_dated_event(:filed, 'date'),
+      make_string_dated_event(:completed, 'claimCompleteDate'),
+      check_for_yes_no_event(:development_letter_sent, 'developmentLetterSent'),
+      check_for_yes_no_event(:decision_notification_sent, 'decisionNotificationSent'),
+      check_for_bool_event(:requested_decision, 'waiver5103Submitted')
+    ]
+
+    # Do the 8 phases
+    (1..8).each { |n|
+      updates << make_string_dated_event("phase#{n}", 'claimPhaseDates', "phase#{n}CompleteDate")
+    }
+
+    # Filter out events that were nil
+    updates.select! { |item| item[:date] }
+
+    # Reverse chron
+    updates.sort! { |a,b| b[:date] <=> a[:date] }
```

Comment:
You could return `nil` from the below methods for any updates that don't have a date, remove L29 and L32 would become something like updates.compact.sort_by{ |h| h[:date] }.reverse